apps/scan/backend/test/helpers/app_helpers.ts

```diff
@@ -133,54 +125,73 @@ export async function waitForStatus(
   }, 1_000);
 }
 
+/**
+ * Creates a mock `PrecinctScannerStateMachine` with no default behavior.
+ */
+export function createPrecinctScannerStateMachineMock(): jest.Mocked<PrecinctScannerStateMachine> {
+  return {
+    status: jest.fn().mockRejectedValue(new Error('not implemented')),
+    scan: jest.fn().mockRejectedValue(new Error('not implemented')),
+    accept: jest.fn().mockRejectedValue(new Error('not implemented')),
+    return: jest.fn().mockRejectedValue(new Error('not implemented')),
+    stop: jest.fn().mockRejectedValue(new Error('not implemented')),
+    calibrate: jest.fn().mockRejectedValue(new Error('not implemented')),
+  };
+}
+
+export async function withApp(
```

Comment:
I intend to replace `createApp` with `withApp` which calls a callback with the values that `createApp` would return in order to add a cleanup step. Because `createApp` starts a server it needs to stop it if we want to avoid the Jest warning about not exiting after the tests finished (see [`--detectOpenHandles`](https://jestjs.io/docs/cli#--detectopenhandles)).